blackbox/docs/admin/system-information.rst

```diff
@@ -1266,6 +1271,13 @@ have corresponding log tables: ``sys.jobs_log`` and ``sys.operations_log``.
 |                              | just by their type.                                |                  |
 +------------------------------+----------------------------------------------------+------------------+
 
+
+.. note::
+
+  You can control which jobs are recorded using the :ref:`Jobs Log Filter
+  Setting <stats.jobs_log_filter>`
```

Comment:
if this is a section title, you don't need to include the link text. if it is a config setting, please include the literal config name